arachni/app/tasks.py

```diff
@@ -0,0 +1,19 @@
+import subprocess
+from celery import Celery
+
+celery = Celery('arachni')
+celery.config_from_object('celeryconfig')
+
+def call_scanner(args):
+    cmd = ' '.join(args)
+    p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+    stdout, stderr = p.communicate()
+    if stdout:
+        stdout = stdout.replace('\n', '<br />')
```

Comment:
The output is plain text, it doesn't contain any html. I can prettify it but don't think it's necessary other than making new lines to look like new lines instead of everything as a single line.